### PR TITLE
Planet index container

### PR DESCRIPTION
**Pull request by eveyv**
worked to refactor code for more efficient and appealing css

### CODE REVIEW

File: app/javascript/react/components/PlanetShow.js [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
-      </div>
-      <div className="planet-tile">
-      {description}
-      </div>
+      <h2>| {name} |</h2>
+        <img className="destination-pic" src={destination_url}/>
+          <div className="planet-description">
+            {description}
+          </div>
     </div>
```
Comment (nwalberts) [inline app/javascript/react/components/PlanetShow.js]: watch your indentation here. please update

File: app/javascript/react/components/PlanetTiles.js [hunk 2]
```diff
@@ -2,15 +2,14 @@ import React from 'react'
...
-  <div className="tiles">
-      <div key={id} className="planet-tile">
-        <Link to={`/planets/${planet.id}`}>| {name} |</Link>
-        <img src={planet_url}/>
+      <div className="cell large-4 medium-6 small-12" key={id}>
+        <Link to={`/planets/${planet.id}`} className="planet-names">| {name} |</Link>
+        <img className="planet-pics" src={planet_url} />
       </div>
-  </div>
+
```
Comment (nwalberts) [inline app/javascript/react/components/PlanetTiles.js]: looks like post fix everything is a bit  indented in here as well.

File: app/javascript/react/components/PlanetsIndexContainer.js [hunk 3]
```diff
@@ -29,17 +29,21 @@ const PlanetsIndexContainer = props => {
...
     return(
-      <PlanetTiles
-        key={planet.id}
-        planet={planet}
-      />
+         <PlanetTiles
+          key={planet.id}
+          planet={planet}
+        />
+
```
Comment (nwalberts) [inline app/javascript/react/components/PlanetsIndexContainer.js]: indentation

File: flexboxgrid-master/.github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+### Actual behavior
+
+...
+
+### Steps to reproduce behavior
+
+* OS:
+* Browser:
+* code/fiddle/pen url:
```
Comment (nwalberts) [inline flexboxgrid-master/.github/ISSUE_TEMPLATE.md]: there are a lot of these flexbox files that should not be installed directly into your root 

Instead, you can get flexbox in your project by adding this to `application.html.erb` 
`<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.min.css" type="text/css" >`

HOWEVER, you shouldnt really even need flexbox, as its patterns are built into Foundation 6! my recommendation would be to remove it altogether

File: package.json [hunk 5]
```diff
@@ -20,18 +16,22 @@
...
+    "enzyme-adapter-react-16": "~1.14.0",
     "fetch-mock": "~5.13.1",
     "jest": "^24.9.0",
-    "enzyme": "~3.10.0",
-    "enzyme-adapter-react-16": "~1.14.0"
+    "webpack-cli": "^2.0.10",
+    "webpack-dev-server": "^2.11.1"
   },
   "scripts": {
     "start": "./bin/webpack-dev-server",
```
Comment (nwalberts) [inline package.json]: remove these changes. While they dont technically change anything, we dont want to change anything having to do with core config

File: spec/controllers/api/v1/planets_controller_spec.rb [hunk 6]
```diff
@@ -12,13 +12,13 @@
...
+      planet_url: 'https://en.wikipedia.org/wiki/Tatooine#/media/File:Tatooine_(fictional_desert_planet).jpg',
       description: 'i dont like sand its coarse and rough',
       ) }
     let!(:second_planet) { Planet.create(
       name: 'Mustafar',
       terrain: 'Lava',
-      url: 'https://vignette.wikia.nocookie.net/starwars/images/d/d1/Hoth.jpg/revision/latest?cb=20071019040631',
+      planet_url: 'https://vignette.wikia.nocookie.net/starwars/images/d/d1/Hoth.jpg/revision/latest?cb=20071019040631',
       description: 'fire burning on the dance floor',
       ) }
```
Comment (nwalberts) [inline spec/controllers/api/v1/planets_controller_spec.rb]: running this spec should break, as you have a trailing comma on line 22. Update accordingly

File: app/javascript/react/components/PlanetsIndexContainer.js [hunk 7]
```diff
@@ -29,17 +29,20 @@ const PlanetsIndexContainer = props => {
...
+      planet={planet}
       />
     )
   })
 
   return(
     <div className="stars">
-      <h2>Explore a galaxy far, far away...</h2>
+      <h1><strong>EXPLORE A GALAXY FAR, FAR AWAY...</strong></h1>
+
```
Comment (safang99) [inline app/javascript/react/components/PlanetsIndexContainer.js]: I would remove this blank line on 42

**Pull request merged**